### DISCUSSION TITLE
List mode data from Philips Vereos PET/CT scanner

### DISCUSSION THREAD

**Discussion by imbeatriz** [villekf/OMEGA#53] [General]
Hi everyone!

I have list mode data from a Philips Vereos PET/CT scanner saved as a .list file. I was wondering if OMEGA could handle this type of data for reconstruction. Are there any available examples of reconstruction using list mode data that you could advice me?

Thank you!

**Comment by villekf** [comment]
Hi

Sorry, Philips listmode data is not supported. I have only experience, and access, to Siemens scanners so I don't have any experience or knowledge of the formats used by other vendors. You'd need to manually get the data out yourself after which you can use OMEGA to reconstruct it, but there isn't an automatic way to get the data out of the listmode file.

If you can extract the listmode data manually, I have some documentation on reconstructing listmode data here: https://omega-doc.readthedocs.io/en/latest/customcoordinates.html

**Reply by imbeatriz** [reply]
Thank you for explaining! It seems that Philips is really hard to get the data out since they don't share any documentation. Do you perhaps have examples of list mode data from Siemens scanners available? An example script would be very helpful, too.

**Reply by villekf** [reply]
The listmode data from the clinical scanners (like Biograph mCT or Vision) are closed-source code, but I do have an example script using the listmode data from the preclinical Siemens Inveon PET scanner: https://github.com/villekf/OMEGA/blob/master/main-files/Inveon_PET_main_listmode_example.m

In case you're using Python (or prefer using it), you can find a similar example here: https://github.com/villekf/OMEGA/blob/master/source/Python/Inveon_PET_main_listmode_example.py

**Reply by imbeatriz** [reply]
Thank you so much for sharing! :)
